chain/network/Cargo.toml

```diff
@@ -47,7 +51,14 @@ adversarial = ["near-network-primitives/adversarial"]
 delay_detector = ["delay-detector"]
 performance_stats = ["near-performance-metrics/performance_stats"]
 sandbox = ["near-network-primitives/sandbox"]
+protocol_feature_routing_exchange_algorithm = []
 
 [[bench]]
 name = "graph"
 harness = false
+
+routing_exchange_algorithm = []
+
+[[bench]]
+name = "ibf"
+harness = false
```

Comment:
@matklad I think the the correct thing to do is to figure out how to run benchmarks in CI rather than convert all of them to `#[test]`

Reply:
Well, even if we run benchmarks in CI we still need a way for it to tell us when something goes wrong. As a result, it makes sense to do what @matklad suggested.

Reply:
I see your point. The main purpose of this code was to get estimates on how much time each part of the code takes.
Rather than adding a hard check and making the code fail if this function runs too slow.

If you really want to make this into a test then in order to avoid having a flunky test, which someones randomly fails, you need to include a factor of 2-3x, you also need to account that someone may be running those tests on a slow laptop machine, that's overheating, which adds another factor of 5-10x. 

I know that this benchmark takes .2ms on my machine, but if you want to make sure tests run everywhere, you wound need to add factor of 20x, so assert that it doesn't take more than 5ms, though tests may still fail randomly, due to how scheduling works in OS when all cores are busy.  That renders type of tests a bit useless.

Anyway, I would like to keep benchmark as they are. They are useful to find out how much time doing certain part of the code takes, and also for optimizing existing code.

If we really want to add those kind of tests, then they need to be under some extra flag, maybe a compiler feature. However, unfortunately, they would be included also under `--all-features`, so that may not be such a good idea.

Reply:
@bowenwang1996 @matklad